instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PostHeader from './PostHeader';
+import PostFooter from './PostFooter';
+import CommentSection from "../CommentSection/CommentSection";
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+
+const PostWrapper = styled.div `
+    max-width: 600px;
+    margin: 20px auto;
+    border: 1px solid lightgrey;
+    background: white;
+    @media (max-width: 650px) {
+        .post-container {
+            margin: 20px 4%; 
+        }
+    }
+`;
+
+const PostImage = styled.img `
+    width: 100%;
+`;
+
+const PostContainer = (props) => {
+    return (
+        <PostWrapper>
+            <PostHeader src={props.data.thumbnailUrl} alt={`${props.data.username} thumbnail`} username={props.data.username} />
```

Comment:
great job passing your props and setting your prop-types here. As you move forward prop-types can save your projects from some odd bugs when working on teams as they won't always know what kind of data type is expected. Prop-types is great at those kinds of situations as it will warn them if they put in the wrong type or forget to add something that is required.